File: laboratorioMombelli/php/L07_AutosaloneDB/src/model/Automobile.php
<?php

namespace APP\model;

class Automobile
{


    public function __construct(private $id, private $marca , private $modello, private $cilindrata , private $prezzo ){}

    public function __get($name){
        return $this->$name;
    }

    public function __set($name, $value){
        $this->$name = $value;
    }

    public function __toString(){
        return "Automobile: id=$this->id, marca=$this->marca, modello=$this->modello, cilindrata=$this->cilindrata, prezzo=$this->prezzo";
    }
}

File: laboratorioMombelli/Java/eclipse-workspace/Prj11_Autosalone/src/model/Automobile.java
package model;

public class Automobile {

	private String marca;
	private String modello;
	private int cilindrata;
	private double prezzo;
	
	// POJO - Plain Old Java Object
	public Automobile() {
		// costruttore vuoto serve a Spring
	}
	
	public Automobile(String marca, String modello, int cilindrata, double prezzo) {
		
		this.marca = marca;
		this.modello = modello;
		this.cilindrata = cilindrata;
		this.prezzo = prezzo;
	}

	public String getMarca() {
		return marca;
	}

	public void setMarca(String marca) {
		this.marca = marca;
	}

	public String getModello() {
		return modello;
	}

	public void setModello(String modello) {
		this.modello = modello;
	}

	public int getCilindrata() {
		return cilindrata;
	}

	public void setCilindrata(int cilindrata) {
		this.cilindrata = cilindrata;
	}

	public double getPrezzo() {
		return prezzo;
	}

	public void setPrezzo(double prezzo) {
		this.prezzo = prezzo;
	}

	@Override
	public String toString() {
		return "Automobile [marca=" + marca + ", modello=" + modello + ", cilindrata=" + cilindrata + ", prezzo="
				+ prezzo + "]";
	}
	
	 
}
